python/3d/auto_generated/symbols/qgsline3dsymbol.sip.in

```diff
@@ -80,13 +80,13 @@ Returns width of the line symbol (in map units)
 Sets width of the line symbol (in map units)
 %End
 
-    float height() const;
+    float offset() const;
```

Comment:
These are API breaks -- you'll need to keep the old ones around for compatibility and mark them as deprecated.

Reply:
`QgsLine3DSymbol` and `QgsPolygon3DSymbol` are considered as unstable API as defined in the header files: "warning This is not considered stable API, and may change in future QGIS releases. It is exposed to the Python bindings as a tech preview only.". 

Do we really want to keep the compatibility?

Reply:
I'd still lean toward compatibility/no breakage in cases where it's trivial to do so.